envi/archs/arm/disasm.py

```diff
@@ -1217,6 +1222,32 @@ def p_uncond(opval, va):
                 shval = (opval>>7) & 0x1f
                 olist = (ArmScaledOffsetOper(Rn, Rm, shtype, shval, va, psize=psize), )
             return (opcode, mnem, olist, 0)
+        elif (opval & 0xff000f0) == 0x5700010:
+            #clrex
+            mnem = "clrex"
+            olist =()
+            #fixme : need opcode
+            opcode = INS_BLX
```

Comment:
how is this an INS_BLX?  if you're going to leave it as anything "meaningless" don't choose a branch :)
better yet, please create INS_CLREX in const.py